wire-ios-data-model/Source/Model/Conversation/ZMConversation+MLS.swift

```diff
@@ -206,6 +202,41 @@ public extension ZMConversation {
     }
 }
 
+// MARK: - Migration releated fetch requests
+
+public extension ZMConversation {
+
+    static func fetchAllTeamGroupProteusConversations(in context: NSManagedObjectContext) throws -> [ZMConversation] {
+        let selfUser = ZMUser.selfUser(in: context)
+        guard let selfUserTeamIdentifier = selfUser.teamIdentifier else {
+            assertionFailure("selfUser.teamIdentifier == nil")
```

Comment:
In itself, the `teamIdentifier` of the self user being `nil` is not considered an error. It's possible that a user is not part of a team (e.g: a user with a personal account instead of a team account). I'd recommend changing the assertion message to make it clearer that we're intentionally crashing because of misuse of the method